documentation/modules/configuring/con-config-node-pools.adoc

```diff
@@ -37,73 +35,72 @@ IMPORTANT: **KRaft mode is not ready for production in Apache Kafka or in Strimz
 
 For a deeper understanding of the node pool configuration options, refer to the link:{BookURLConfiguring}[Strimzi Custom Resource API Reference^].
 
-NOTE: While the `KafkaNodePools` feature gate that enables node pools is in alpha phase, replica and storage configuration properties in the `KafkaNodePool` resource must also be present in the `Kafka` resource. The configuration in the `Kafka` resource is ignored when node pools are used. Similarly, ZooKeeper configuration properties must also be present in the `Kafka` resource when using KRaft mode. These properties are also ignored.
+NOTE: While the `KafkaNodePools` feature gate that enables node pools is in beta phase, replica and storage configuration properties in the `KafkaNodePool` resource must also be present in the `Kafka` resource. The configuration in the `Kafka` resource is ignored when node pools are used. Similarly, ZooKeeper configuration properties must also be present in the `Kafka` resource when using KRaft mode. These properties are also ignored.
 
-.Example configuration for a node pool in a cluster using ZooKeeper
+.Example configuration for a node pool in a cluster using KRaft mode 
 [source,yaml,subs="+attributes"]
 ----
 apiVersion: {KafkaNodePoolApiVersion}
 kind: KafkaNodePool
 metadata:
-  name: pool-a # <1>
+  name: kraft-dual-role
   labels:
-    strimzi.io/cluster: my-cluster # <2>
+    strimzi.io/cluster: my-cluster
 spec:
-  replicas: 3 # <3>
-  roles:
-    - broker # <4>
-  storage: # <5>
+  replicas: 3
+  roles: # <1>
+    - controller
+    - broker
+  storage:
     type: jbod
     volumes:
       - id: 0
         type: persistent-claim
-        size: 100Gi
+        size: 20Gi
```

Comment:
Why change the volume size? I do not see any value and 20Gi is in general quite small. I also wonder if we should keep the callouts for the other parts.

Reply:
That's peculiar. I just changed the order of the examples so that KRaft came first.  I didn't make any changes to the examples themselves. I'll revert this.

Reply:
Ahh, I did not realize that. But especially for the callouts, it makes sense to have them in the first example rather than the second. It looks good now.